src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -20,40 +20,69 @@ namespace Nethermind.JsonRpc
+            DefaultValue = "false")]
         bool RpcRecorderEnabled { get; set; }
 
-        [ConfigItem(Description = "Port number for JSON RPC calls. Ensure the firewall is configured when enabling JSON RPC.", DefaultValue = "8545")]
+        [ConfigItem(
+            Description = "Port number for JSON RPC calls. Ensure the firewall is configured when enabling JSON RPC.",
+            DefaultValue = "8545")]
         int Port { get; set; }
         
-        [ConfigItem(Description = "Port number for JSON RPC web sockets calls. By default same port is used as regular JSON RPC. Ensure the firewall is configured when enabling JSON RPC.", DefaultValue = "8545")]
+        [ConfigItem(
+            Description = "Port number for JSON RPC web sockets calls. By default same port is used as regular JSON RPC. Ensure the firewall is configured when enabling JSON RPC.",
+            DefaultValue = "8545")]
         int WebSocketsPort { get; set; }
         
-        [ConfigItem(Description = "Defines which RPC modules should be enabled.", DefaultValue = "all")]
+        [ConfigItem(
+            Description = "Defines which RPC modules should be enabled.",
+            DefaultValue = "all")]
         string[] EnabledModules { get; set; }
         
-        [ConfigItem(Description = "Defines block depth when finding logs.", DefaultValue = "1000")]
+        [ConfigItem(
+            Description = "Defines block depth when finding logs.",
+            DefaultValue = "1000")]
         int FindLogBlockDepthLimit { get; set; }
         
-        [ConfigItem(Description = "Gas limit for eth_call and eth_estimateGas", DefaultValue = "100000000")]
+        [ConfigItem(
+            Description = "Gas limit for eth_call and eth_estimateGas",
+            DefaultValue = "100000000")]
         long? GasCap { get; set; }
         
-        [ConfigItem(Description = "Interval between the JSON RPC stats report log", DefaultValue = "300")]
+        [ConfigItem(
+            Description = "Interval between the JSON RPC stats report log",
+            DefaultValue = "300")]
         public int ReportIntervalSeconds { get; set; }
         
-        [ConfigItem(Description = "Buffer responses before sending them to client. This allows to set Content-Length in response instead of using Transfer-Encoding: chunked. This may degrade performance on big responses.", DefaultValue = "false")]
+        [ConfigItem(
+            Description = "Buffer responses before sending them to client. This allows to set Content-Length in response instead of using Transfer-Encoding: chunked. This may degrade performance on big responses.",
+            DefaultValue = "false")]
         public bool BufferResponses { get; set; }
+        
+        [ConfigItem(
+            Description = "A path to a file that contains a list of new-line separated approved JSON RPC calls",
+            DefaultValue = "Data/jsonrpc.filter")]
+        public string CallsFilterFilePath { get; set; }
```

Comment:
I don't like its only file-based configuration and not supporting also simple parameter (regex)

Reply:
prefer simplicity